boards/omote/README.md

```diff
@@ -0,0 +1,73 @@
+Oppila Microsystems omote platform -  http://www.oppila.in
```

Comment:
Please move the content of the README.md file to doc.txt. It contains valuable information to have in the generated documentation.
Once its content is moved, simply remove the readme.